bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/controller/BusinessPartnerController.kt

```diff
@@ -66,4 +69,22 @@ class BusinessPartnerController(
     ): PageDto<BusinessPartnerOutputDto> {
         return businessPartnerService.getBusinessPartnersOutput(paginationRequest.toPageRequest(), externalIds,  getCurrentUserBpn())
     }
+
+    @PreAuthorize("hasAuthority(${PermissionConfigProperties.WRITE_INPUT_PARTNER})")
```

Comment:
see REST API conformity:
Best each resource has its own permissions for each operation. read for GET and create/write for POST

For this, the applicationConfig needs to be adjusted. Also the default realm files need to be adjusted. (Ideally in both Helm Charts and Tests)